From: Stuart Cardall <developer@it-offshore.co.uk>
Date: Fri, 10 Feb 2017 15:26:30 +0000
Subject: [PATCH] fix help message

Adds -u option to help message.

Alpine notes:
* This patch is ported from upstream commit:
  https://github.com/wojtekka/6tunnel/commit/f800eb1fe5b54ad1022c1bebd2a41a650b86752d

index c33865a..b8d551e 100644
--- a/6tunnel.c
+++ b/6tunnel.c
@@ -460,6 +460,7 @@
 "  -d  don't detach\n"
 "  -f  force tunneling (even if remotehost isn't resolvable)\n"
 "  -h  print hex dump of packets\n"
+"  -u  change UID and GID after bind()\n"
 "  -i  act like irc proxy and ask for password\n"
 "  -I  send specified password to the irc server\n"
 "  -l  bind to specified address\n"
